nixos/modules/services/network-filesystems/webdav.nix

```diff
@@ -0,0 +1,91 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.services.webdav;
+in
+{
+  options = {
+    services.webdav = {
+      enable = mkEnableOption "WebDAV server";
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.webdav;
+        description = "The WebDAV package to use.";
+      };
```

Comment:
Would a sysadmin ever change this value?

Reply:
Providing an option to change service package is always a good thing, because it makes it easier to try a different package version and test a patch.

Reply:
I know nix overlay.  I am coming from the Guix world and a Guix service always provides an option to override package.  I'll remove this option if using overlay here is the standard nix way.